### ISSUE TITLE
Png images contain random noise pixels.

### ISSUE THREAD

**Issue by Xhilbert** [wkhtmltopdf/wkhtmltopdf#4267]
When using wkhtmltoimage with png and transparent options, there are random noise pixels in the png file created.

It has been descriped in the bug  #2629.

franjaviersans found the solution !

**"Qt images are initialized with uninitialized data, as stated in -> http://doc.qt.io/qt-5/qimage.html#QImage-1

A line with "image.fill(qRgba(0, 0, 0, 0));" should be added after the initialization of the image to ensure that some data will always be on the background. This will fix issues with images with transparent background."**

Could you please fix this bug, and put the new release version on github ?

Thank you very much.

**Comment by PhilterPaper** [comment]
Why is this a new ticket, rather than adding your request to #2629?

**Comment by Xhilbert** [comment]
The ticket #2629 was posted in 2015 and I feared that the developpers forget it.
The solution to fix this bug was written in the old post and nobody reacted, so I tried my chance with this new ticket because it would help me a lot if this bug was solved.
Sorry for that, and thank you for your question.

**Comment by PhilterPaper** [comment]
Adding to #2629 would bump that thread to the top of the ticket list, so it shouldn't be forgotten. Whether anyone feels they have a handle on the problem and might work on it, that's the question. Perhaps they feel that the given solution is not a good one? If so, it would be nice for someone to comment _why_ they feel that fix is no good, so others could take a cut at it.

**Comment by Xhilbert** [comment]
I didn't know adding to #2629 would bump that thread to the top. Thank you.
I think the given solution is good because the symptoms I can see match perfectly with the explanation 
franjaviersans. (when I use twice "wkhtmltoimage_convert" in the same programm, I can see the remaining pixels of the last time I called it, even though I used "wkhtmltoimage_destroy_converter" and I created a new one !)
This bug should be fixed quite easily for someone who knows where to add the line suggested by 
franjaviersans. I don't know how to build dll from the wkhtmlmaster project, and maybe people who saw this bug know how to. That could explain why people don't complain about it anymore.
